plugins/vlc-video/vlc-video-source.c

```diff
@@ -417,9 +419,12 @@ static void add_file(struct vlc_source *c, struct darray *array,
 		new_media = create_media_from_file(path);
 
 	if (new_media) {
-		if (is_url)
-			libvlc_media_add_option_(new_media,
-					":network-caching=100");
+		if (is_url) {
+			struct dstr network_caching_option = {0};
+			dstr_catf(&network_caching_option, ":network-caching=%d", network_caching);
+			libvlc_media_add_option_(new_media, network_caching_option.array);
```

Comment:
This line goes over the 80 character limit.  Insert a line break after "new_media," (right after the comma), delete the proceeding space character (right before "network_caching_option.array"), and insert two tabs to indent.